composer.json

```diff
@@ -40,5 +40,8 @@
         "psr-4": {
             "Psl\\Tests\\": "tests/"
         }
+    },
+    "scripts": {
+        "phpunit": "phpunit --coverage-text"
```

Comment:
I usually don't add ./vendor/bin/phpunit because composer does it for me see [custom command doc](https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands)

Reply:
That's awesome! I didn't know that. Nice!